maps/HDRUK/3.0.0/GWDM/2.0/translation.jsonata

```diff
@@ -18,12 +18,11 @@ $nullString := function($x){
 };
 
 $nullList := function($x){
-    [$map($x, function($xx){
-        $filter($nullString($xx), function($item){ $item != null })
+    [$filter($x, function($xx){
+        $map($nullString($xx), function($item){ $item != null })
```

Comment:
noted this is a bug, i think i wrote this function for another translation map?

Reply:
I checked and this is the only file where this function appears (by this name at least).